### PR TITLE
[clang-tidy] concatenate nested namespaces

### PR DESCRIPTION
**Pull request by neheb**
Found with modernize-concat-nested-namespaces

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: src/session_impl.cpp [hunk 1]
```diff
@@ -170,20 +170,7 @@ namespace boost {
...
-#if defined TORRENT_ASIO_DEBUGGING
-	std::map<std::string, async_t> _async_ops;
-	std::deque<wakeup_t> _wakeups;
-	int _async_ops_nthreads = 0;
-	std::mutex _async_ops_mutex;
-
-	std::map<int, handler_alloc_t> _handler_storage;
-	std::mutex _handler_storage_mutex;
-	bool _handler_logger_registered = false;
-#endif
```
Comment (arvidn) [inline src/session_impl.cpp:184]: what happened here?

Comment (arvidn) [reply src/session_impl.cpp:184]: what I mean is; this should not be removed

**Comment by arvidn** [comment]
I would prefer `modernize-concat-nested-namespaces` be added to clang-tidy as well

File: src/session_impl.cpp (hunk 1 above)
Comment (neheb) [reply src/session_impl.cpp:184]: Looks like clang-tidy goofed. I'll manually fix.

**Comment by arvidn** [comment]
https://travis-ci.org/github/arvidn/libtorrent/jobs/704512527#L836

**Comment by arvidn** [comment]
This one is in a separate repository
https://travis-ci.org/github/arvidn/libtorrent/jobs/704512527#L1024

**Comment by neheb** [comment]
What is this try_signal?

**Comment by arvidn** [comment]
https://github.com/arvidn/try_signal

**Comment by glassez** [comment]
Sorry, isn't "nested namespace declaration" available since c++17? Or libtorrent master no longer support c++14?

**Comment by arvidn** [comment]
master is C++17
2.0 is C++14
1.2.x is C++11

**Pull request closed without merging**